### PR TITLE
Translate 2.6.0-rc{1,2} release (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Ref: https://github.com/ruby/www.ruby-lang.org/issues/1894

This translates https://github.com/ruby/www.ruby-lang.org/pull/1909, https://github.com/ruby/www.ruby-lang.org/pull/1920

Check cbf224f6 for review, @ruby/www-ruby-lang-org-i18n-ko :pray:

### CODE REVIEW

**Comment by yous** [comment]
There is a new commit for 2.6.0-rc2: https://github.com/ruby/www.ruby-lang.org/commit/f8e7266e51d4ff00e3456d38a58c9f69cb944dfd

**Comment by riseshia** [comment]
35862d89 Add follow up of f8e7266

File: ko/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+  종료 지정이 없는 범위 연산자, `(1..)`가 추가됩니다. 이는 끝이 없는 것처럼 취급됩니다. 다음은 전형적인 사용 예시입니다.
+
+      ary[1..]                          # ary[1..-1]와 동치
+      (1..).each {|index| ... }         # 1로 시작하는 무한 루프
+      ary.zip(1..) {|elem, index| ... } # ary.each.with_index(1) { ... }
+
+* `Binding#source_location`을 추가했습니다. [[Feature #14230]](https://bugs.ruby-lang.org/issues/14230)
+
+  이 메서드는 `binding`의 소스 코드 상의 위치를 `__FILE__`과 `__LINE__`을 가지는 배열로 돌려줍니다. `Kernel#eval`이 `binding`의 소스 코드의 위치를 무시하도록 변경할 예정입니다. [[Bug #4352]](https://bugs.ruby-lang.org/issues/4352) 그러므로 지금까지 사용하던 `eval("[__FILE__, __LINE__]", binding)`로 같은 정보를 획득할 수 없게 됩니다,  앞으로는 `Kernel#eval`보다는 새로운 `Binding#source_location` 메서드를 사용하게 될 것입니다.
```
Comment (yous) [inline ko/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md]:
```suggestion
  이 메서드는 `binding`의 소스 코드 상의 위치를 `__FILE__`과 `__LINE__`을 가지는 배열로 돌려줍니다. `Kernel#eval`이 `binding`의 소스 코드의 위치를 무시하도록 변경할 예정입니다. [[Bug #4352]](https://bugs.ruby-lang.org/issues/4352) 그러므로 지금까지 사용하던 `eval("[__FILE__, __LINE__]", binding)`로 같은 정보를 획득할 수 없게 됩니다. 앞으로는 `Kernel#eval`보다는 새로운 `Binding#source_location` 메서드를 사용하게 될 것입니다.
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+RC2는 Bundler 2.0 대신 1.17을 포함해보기 위해서 릴리스되었습니다.
+이는 다음과 같은 여러 새 기능과 기능 향상을 포함합니다.
+
+## JIT
+
+루비 2.6은 JIT(Just-in-time) 컴파일러의 첫 구현체를 포함합니다.
+
+JIT 컴파일러는 루비 프로그램의 실행 성능을 향상시키는 것이 목적입니다.
+다른 언어의 일반적인 JIT 컴파일러와는 다르게, 루비의 JIT 컴파일러는 C 코드를 디스크에 출력한 뒤, 일반적인 C 컴파일러 프로세스를 사용해 네이티브 코드를 생성하도록 합니다.
+다음을 참고하세요. [(Vladimir Makarov가) 작성한 MJIT 구조](https://github.com/vnmakarov/ruby/tree/rtl_mjit_branch#mjit-organization).
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]: How about just removing parentheses?
```suggestion
[Vladimir Makarov가 작성한 MJIT 구조](https://github.com/vnmakarov/ruby/tree/rtl_mjit_branch#mjit-organization)를 참고하세요.
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+2.6.0-rc2에서는 Optcarrot이라는 CPU 성능을 요구하는 벤치마크에서 1.7배의 성능 향상을 이루었습니다(다음을 참조: https://gist.github.com/k0kubun/d7f54d96f8e501bbbc78b927640f4208). Rails 애플리케이션 같은 메모리를 요구하는 작업에서도 성능을 향상시킬 것입니다.
+
+새로운 루비의 성능을 기대해주세요.
+
+## RubyVM::AbstractSyntaxTree [Experimental]
+
+루비 2.6에는 `RubyVM::AbstractSyntaxTree` 모듈이 도입되었습니다.
+
+이 모듈에은 문자열을 파싱하여 AST(추상구문트리)의 노드를 돌려주는 `parse` 메서드, 파일을 파싱하여 AST의 노드를 돌려주는 `parse_file` 메서드가 들어있습니다.
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]:
```suggestion
이 모듈에는 문자열을 파싱하여 AST(추상구문트리)의 노드를 돌려주는 `parse` 메서드, 파일을 파싱하여 AST의 노드를 돌려주는 `parse_file` 메서드가 들어있습니다.
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+`RubyVM::AbstractSyntaxTree::Node`도 도입되었습니다. 이 클래스의 인스턴스로부터 위치정보나 자식 노드를 얻을 수 있습니다. 이 기능은 실험적으로 포함되었으며, AST 노드의 구조는 호환성을 보장하지 않습니다.
+
+## 새로운 기능
+
+* `Kernel#yield_self`의 별칭으로 `then`이 추가되었습니다. [[Feature #14594]](https://bugs.ruby-lang.org/issues/14594)
+
+* `rescue`가 없는 `else`가 문법 에러가 됩니다. [EXPERIMENTAL]
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]:
```suggestion
* `rescue`가 없는 `else`가 문법 에러가 됩니다. [EXPERIMENTAL] [[Feature #14606]](https://bugs.ruby-lang.org/issues/14606)
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+* ASCII 이외의 대문자로 시작하는 상수를 정의할 수 있게 됩니다. [[Feature #13770]](https://bugs.ruby-lang.org/issues/13770)
+
+* 종료 지정이 없는 범위 연산자가 추가됩니다. [[Feature #12912]](https://bugs.ruby-lang.org/issues/12912)
+
+  종료 지정이 없는 범위 연산자, `(1..)`가 추가됩니다. 이는 끝이 없는 것처럼 취급됩니다. 다음은 전형적인 사용 예시입니다.
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]:
```suggestion
  종료 지정이 없는 범위 연산자 `(1..)`가 추가됩니다. 이는 끝이 없는 것처럼 취급됩니다. 다음은 전형적인 사용 예시입니다.
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+      ary[1..]                          # ary[1..-1]와 동치
+      (1..).each {|index| ... }         # 1로 시작하는 무한 루프
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]:
```suggestion
      (1..).each {|index| block }          # 1로 시작하는 무한 루프
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+      ary.zip(1..) {|elem, index| ... } # ary.each.with_index(1) { ... }
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]:
```suggestion
      ary.zip(1..) {|elem, index| block }  # ary.each.with_index(1) { }
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+
+* `rescue`가 없는 `else`가 문법 에러가 됩니다. [EXPERIMENTAL]
+
+* ASCII 이외의 대문자로 시작하는 상수를 정의할 수 있게 됩니다. [[Feature #13770]](https://bugs.ruby-lang.org/issues/13770)
+
+* 종료 지정이 없는 범위 연산자가 추가됩니다. [[Feature #12912]](https://bugs.ruby-lang.org/issues/12912)
+
+  종료 지정이 없는 범위 연산자, `(1..)`가 추가됩니다. 이는 끝이 없는 것처럼 취급됩니다. 다음은 전형적인 사용 예시입니다.
+
+      ary[1..]                          # ary[1..-1]와 동치
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]:
```suggestion
      ary[1..]                             # ary[1..-1]와 동치
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+      (1..).each {|index| ... }         # 1로 시작하는 무한 루프
+      ary.zip(1..) {|elem, index| ... } # ary.each.with_index(1) { ... }
+
+* `Binding#source_location`을 추가했습니다. [[Feature #14230]](https://bugs.ruby-lang.org/issues/14230)
+
+  이 메서드는 `binding`의 소스 코드 상의 위치를 `__FILE__`과 `__LINE__`을 가지는 배열로 돌려줍니다. `Kernel#eval`이 `binding`의 소스 코드의 위치를 무시하도록 변경할 예정입니다. [[Bug #4352]](https://bugs.ruby-lang.org/issues/4352) 그러므로 지금까지 사용하던 `eval("[__FILE__, __LINE__]", binding)`로 같은 정보를 획득할 수 없게 됩니다,  앞으로는 `Kernel#eval`보다는 새로운 `Binding#source_location` 메서드를 사용하게 될 것입니다.
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]:
```suggestion
  이 메서드는 `binding`의 소스 코드 상의 위치를 `__FILE__`과 `__LINE__`을 가지는 배열로 돌려줍니다. `Kernel#eval`이 `binding`의 소스 코드의 위치를 무시하도록 변경할 예정입니다. [[Bug #4352]](https://bugs.ruby-lang.org/issues/4352) 그러므로 지금까지 사용하던 `eval("[__FILE__, __LINE__]", binding)`로 같은 정보를 획득할 수 없게 됩니다. 앞으로는 `Kernel#eval`보다는 새로운 `Binding#source_location` 메서드를 사용하게 될 것입니다.
```

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+* `Kernal#system`이 실패했을 경우 `false`를 돌려주는 대신, 에러를 던지도록 하는 `:exception` 옵션을 추가했습니다. [[Feature #14386]](https://bugs.ruby-lang.org/issues/14386)
+
+* `Coverage`의 oneshot_lines 모드를 추가했습니다. [[Feature#15022]](https://bugs.ruby-lang.org/issues/15022)
+
+  * 이 모드는 '각 줄이 몇 번 실행되었는지' 대신 '각 줄이 한 번 이상 실행되었는지'를 확인합니다. 각 줄의 훅은 최대 1회만 실행되며, 실행된 후에는 플래그를 제거하기 때문에 오버헤드 없이 실행됩니다.
+  * `Coverage.start`에 `:oneshot_lines` 키워드 인수가 추가됩니다.
+  * `Coverage.result`에 `:stop`과 `:clear` 키워드 인수가 추가됩니다. 만약 `clear`가 참이라면, 이는 카운터를 0으로 초기화합니다. 만약 `stop`이 참이라면 커버리지 측정을 비활성화합니다.
+  * 주어진 소스 코드로부터 'stub'을 생성하는 간단한 헬퍼 함수인 `Coverage.line_stub`을 추가합니다.
+
+* `FileUtils#cp_lr`을 추가했습니다.  [[Feature #4189]](https://bugs.ruby-lang.org/issues/4189)
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]:
```suggestion
* `FileUtils#cp_lr`을 추가했습니다. [[Feature #4189]](https://bugs.ruby-lang.org/issues/4189)
```

File: ko/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+  가지는 메모리 객체들을 관리합니다. 예를 들어 작고 짧게 생존하는 Hash 객체는
+  2배 빨라집니다. rdoc 벤치마크에서 6-7%의 성능 향상을 확인했습니다.
+
+## 2.5 이후 주목할 만한 변경
+
+* `$SAFE`가 프로세스 전역 변수로 취급되며, `0` 이외의 값을 설정한 후에 `0`으로 되돌리는 것이 가능해집니다. [[Feature #14250]](https://bugs.ruby-lang.org/issues/14250)
+
+* `ERB.new`에 `safe_level`을 넘기는 기능이 제거될 예정입니다. 또한 `trim_mode`와 `eoutvar`는 키워드 변수로 변경됩니다. [[Feature #14256]](https://bugs.ruby-lang.org/issues/14256)
+
+* 유니코드 지원 버전이 11.0.0으로 갱신되었습니다. 이는 루비 2.6의 TEENY 릴리스에서 12와 12.1로 갱신될 예정입니다.
```
Comment (yous) [inline ko/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md]: Is there a reason to use `11.0.0` here? The original one is `11`.

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+  가지는 메모리 객체들을 관리합니다. 예를 들어 작고 짧게 생존하는 Hash 객체는
+  2배 빨라집니다. rdoc 벤치마크에서 6-7%의 성능 향상을 확인했습니다.
+
+## 2.5 이후 주목할 만한 변경
+
+* `$SAFE`가 프로세스 전역 변수로 취급되며, `0` 이외의 값을 설정한 후에 `0`으로 되돌리는 것이 가능해집니다. [[Feature #14250]](https://bugs.ruby-lang.org/issues/14250)
+
+* `ERB.new`에 `safe_level`을 넘기는 기능이 제거될 예정입니다. 또한 `trim_mode`와 `eoutvar`는 키워드 변수로 변경됩니다. [[Feature #14256]](https://bugs.ruby-lang.org/issues/14256)
+
+* 유니코드 지원 버전이 11.0.0으로 갱신되었습니다. 이는 루비 2.6의 TEENY 릴리스에서 12와 12.1로 갱신될 예정입니다.
```
Comment (yous) [inline ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]: `11.0.0` is used here too.

File: ko/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+루비 2.6.0-rc1은 루비 2.6.0의 첫 번째 릴리스 후보입니다.
+이는 다음과 같은 여러 새 기능과 기능 향상을 포함합니다.
+
+## JIT
+
+루비 2.6은 JIT(Just-in-time) 컴파일러의 첫 구현체를 포함합니다.
+
+JIT 컴파일러는 루비 프로그램의 실행 성능을 향상시키는 것이 목적입니다.
+다른 언어의 일반적인 JIT 컴파일러와는 다르게, 루비의 JIT 컴파일러는 C 코드를 디스크에 출력한 뒤, 일반적인 C 컴파일러 프로세스를 사용해 네이티브 코드를 생성하도록 합니다.
+다음을 참고하세요. [Vladimir Makarov가 작성한 MJIT 구조](https://github.com/vnmakarov/ruby/tree/rtl_mjit_branch#mjit-organization).
```
[the remark this answers is not in the document]
Comment (riseshia) [reply ko/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md]: 👍

File: ko/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md (hunk 11 above)
Comment (riseshia) [reply ko/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md]: ade1b1d8 Oops... nice catch!

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md (hunk 12 above)
Comment (riseshia) [reply ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]: Fixed https://github.com/ruby/www.ruby-lang.org/pull/1924#discussion_r243892932

File: ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md [hunk 14]
```diff
@@ -0,0 +1,139 @@
...
+  가지는 메모리 객체들을 관리합니다. 예를 들어 작고 짧게 생존하는 Hash 객체는
+  2배 빨라집니다. rdoc 벤치마크에서 6-7%의 성능 향상을 확인했습니다.
+
+## 2.5 이후 주목할 만한 변경
+
+* `$SAFE`가 프로세스 전역 변수로 취급되며, `0` 이외의 값을 설정한 후에 `0`으로 되돌리는 것이 가능해집니다. [[Feature #14250]](https://bugs.ruby-lang.org/issues/14250)
+
+* `ERB.new`에 `safe_level`을 넘기는 기능이 제거될 예정입니다. 또한 `trim_mode`와 `eoutvar`는 키워드 변수로 변경됩니다. [[Feature #14256]](https://bugs.ruby-lang.org/issues/14256)
+
+* 유니코드 지원 버전이 11으로 갱신되었습니다. 이는 루비 2.6의 TEENY 릴리스에서 12와 12.1로 갱신될 예정입니다.
```
[the remark this answers is not in the document]
Comment (riseshia) [reply ko/news/_posts/2018-12-15-ruby-2-6-0-rc2-released.md]: 🙈

**Pull request merged**